pkg/crc/ssh/keys_windows.go

```diff
@@ -31,13 +31,16 @@ func (kp *KeyPair) WriteToFile(privateKeyPath string, publicKeyPath string) erro
 		}
 
 		if _, err := f.Write(v.Value); err != nil {
+			_ = f.Close()
```

Comment:
You could use `defer` to avoid the need to put this in all code paths, especially as you are ignoring the error from `Close`:
```
		f, err := os.Create(v.File)
		if err != nil {
			return ErrUnableToWriteFile
		}
		defer f.Close()
```

However, see https://www.sobyte.net/post/2022-01/golang-defer-file-close/ regarding `Close()` errors.
If I remember correctly, what we do in other parts of crc is:
```
		f, err := os.Create(v.File)
		if err != nil {
			return ErrUnableToWriteFile
		}
		defer f.Close()
 		//more code
		return f.Close() // instead of return nil
```

Reply:
Amended the commit to use `defer f.Close()`